src/utils/Validations.ts

```diff
@@ -73,3 +77,35 @@ export const isPersonNameValid = (personName: PersonIdentityName[]) : boolean =>
 export const isValidUUID = (code: string): boolean => {
 	return Constants.REGEX_UUID.test(code);
 };
+export const eventToSubjectIdentifier = async (jwtAdapter: KmsJwtAdapter, event: APIGatewayProxyEvent): Promise<string> => {
```

Comment:
I like this function! Could we please have some tests for it?

Reply:
Tests added